lib/Agrammon/Web/Service.pm6

```diff
@@ -118,13 +121,32 @@ class Agrammon::Web::Service {
                         technical => %!technical-parameters;
             }
         }
+    }
+
+    method get-output-variables(Agrammon::Web::SessionUser $user, Str $dataset-name) {
+        my $outputs = self!get-outputs($user, $dataset-name);
 
         # TODO: get with-filters from frontend
         my %gui-output = output-for-gui($!model, $outputs, True);
         warn '**** get-output-variables() not yet completely implemented';
         return %gui-output;
     }
 
+    method get-excel-export(Agrammon::Web::SessionUser $user, %params) {
+        my $dataset-name = %params<datasetName>;
+        my $language     = %params<language>;
+        my $with-filters = %params<withFilters>;
+
+        my %inputs; # TODO: handle inputs
+        my $outputs = self!get-outputs($user, $dataset-name);
+        my $export = output-as-excel(
+            $dataset-name,
+            $!model, $outputs, $language,
+            $with-filters
+        );
+        return $export;
```

Comment:
Don't really need this variable :-)

Reply:
This was left over from debugging ... removed